gdx/src/com/badlogic/gdx/maps/tiled/BaseTmjMapLoader.java

```diff
@@ -0,0 +1,746 @@
+				offsetY = offset.getInt("y", 0);
+			}
+			TiledMapTileSet tileSet = new TiledMapTileSet();
+
+			// TileSet
+			tileSet.setName(name);
+			final MapProperties tileSetProperties = tileSet.getProperties();
+			JsonValue properties = element.get("properties");
+			if (properties != null) {
+				loadProperties(tileSetProperties, properties);
+			}
+			tileSetProperties.put("firstgid", firstgid);
+
+			// Tiles
+			JsonValue tiles = element.get("tiles");
+
+			if (tiles == null) {
+				tiles = new JsonValue(JsonValue.ValueType.array);
+			}
+
+			addStaticTiles(tmjFile, imageResolver, tileSet, element, tiles, name, firstgid, tilewidth, tileheight, spacing, margin,
+				source, offsetX, offsetY, imageSource, imageWidth, imageHeight, image);
+
+			Array<AnimatedTiledMapTile> animatedTiles = new Array<>();
+
+			for (JsonValue tileElement : tiles) {
+				int localtid = tileElement.getInt("id", 0);
+				TiledMapTile tile = tileSet.getTile(firstgid + localtid);
+				if (tile != null) {
+					AnimatedTiledMapTile animatedTile = createAnimatedTile(tileSet, tile, tileElement, firstgid);
+					if (animatedTile != null) {
+						animatedTiles.add(animatedTile);
+						tile = animatedTile;
+					}
+					addTileProperties(tile, tileElement);
+					addTileObjectGroup(tile, tileElement);
+				}
+			}
+
+			// replace original static tiles by animated tiles
+			for (AnimatedTiledMapTile animatedTile : animatedTiles) {
+				tileSet.putTile(animatedTile.getId(), animatedTile);
+			}
+
+			map.getTileSets().addTileSet(tileSet);
+		}
+	}
+
+	protected abstract void addStaticTiles (FileHandle tmjFile, ImageResolver imageResolver, TiledMapTileSet tileSet,
+		JsonValue element, JsonValue tiles, String name, int firstgid, int tilewidth, int tileheight, int spacing, int margin,
+		String source, int offsetX, int offsetY, String imageSource, int imageWidth, int imageHeight, FileHandle image);
+
+	private void addTileProperties (TiledMapTile tile, JsonValue tileElement) {
+		String terrain = tileElement.getString("terrain", null);
+		if (terrain != null) {
+			tile.getProperties().put("terrain", terrain);
+		}
+		String probability = tileElement.getString("probability", null);
+		if (probability != null) {
+			tile.getProperties().put("probability", probability);
+		}
+		String type = tileElement.getString("type", null);
+		if (type != null) {
+			tile.getProperties().put("type", type);
+		}
+		JsonValue properties = tileElement.get("properties");
+		if (properties != null) {
+			loadProperties(tile.getProperties(), properties);
+		}
+	}
+
+	private void addTileObjectGroup (TiledMapTile tile, JsonValue tileElement) {
+		JsonValue objectgroupElement = tileElement.get("objectgroup");
+		if (objectgroupElement != null) {
+			for (JsonValue objectElement : objectgroupElement.get("objects")) {
+				loadObject(this.map, tile, objectElement);
+			}
+		}
+	}
+
+	private AnimatedTiledMapTile createAnimatedTile (TiledMapTileSet tileSet, TiledMapTile tile, JsonValue tileElement,
```

Comment:
There was also an issue here in the createAnimatedTile() method. When parsing animated tiles as there is no "frame" key under the animation element to look for in the json file, which is different than the .tmx files.
Here's what the animation portion contains:
"animation":[
                {
                 "duration":100,
                 "tileid":176
                }],
         "id":184
        }, 

I will provide a working version of the method below:
```
private AnimatedTiledMapTile createAnimatedTile (TiledMapTileSet tileSet, TiledMapTile tile, JsonValue tileElement,
		 int firstgid) {
		  JsonValue animationElement = tileElement.get("animation");
		  if (animationElement != null) {
				Array<StaticTiledMapTile> staticTiles = new Array<>();
				IntArray intervals = new IntArray();
				for (JsonValue frameValue : animationElement) {
					 staticTiles.add((StaticTiledMapTile)tileSet.getTile(firstgid + frameValue.getInt("tileid")));
					 intervals.add(frameValue.getInt("duration"));
				}

				AnimatedTiledMapTile animatedTile = new AnimatedTiledMapTile(intervals, staticTiles);
				animatedTile.setId(tile.getId());
				return animatedTile;
		  }
		  return null;
	 }
```